python/mxnet/io.py

```diff
@@ -577,8 +577,16 @@ class MXDataIter(DataIter):
 
```

Comment:
"this outputs one data blob and one label blob" seems to already get taken care of in the `DataIter` documentation. I guess no need to repeat it in the `MXDataIter` doc. Let me know if there are other things I should explain. Thanks!!